### DISCUSSION TITLE
What's the deal with skip_ui_tests?

### DISCUSSION THREAD

**Discussion by dmit** [time-rs/time#344] [Question] [answered]
It appears that those are only run on Ubuntu right now in CI. Also looks like only the `compile-fail` subset of integration tests is affected. How come these tests fail on non-Ubuntu platforms?

**Comment by jhpratt** [comment, marked as answer]
IIRC it's because at one point I had `.stderr` files, so the paths would be slightly different, thus failing CI. I don't remember if `compilefail-rs` works on all platforms either.

It's probably best to keep it this way, as I intend to switch to the `trybuild` crate once it doesn't completely ruin the code coverage analysis.
